python/kvikio/__init__.py

```diff
@@ -1,8 +1,7 @@
 # Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
 # See file LICENSE for terms.
 
-
-from . import _version
+from . import _version, nvcomp  # noqa: F401
```

Comment:
```suggestion
from . import _version
```
No need to import the module always.

Reply:
@thomcom I insist :) 
Let's keep implicit imports to a minimum.

Reply:
Haha! It's your library!